app/models/invoice.rb

```diff
@@ -1,4 +1,5 @@
 class Invoice < ApplicationRecord
+  default_scope { order('id ASC') }
```

Comment:
@case-eee You are right. We took care of this already, no more `default scope` for `Invoice`